README.md

```diff
@@ -48,17 +48,18 @@ Step 1: Clone the repository
 Step 2: Download the datasets and move the data files to the dataset directory
 
   ```sh
-  wget -r -np -R "index.html*"  http://43.138.115.238/download/febench/data/; cp -r <dataset directory> ./dataset
+  wget -r -np -R "index.html*"  http://43.138.115.238/download/data/ -P ./dataset
   ```
 
 > Note the data files are in parquet format.
 
-Step 3: Start the cluster and enter the *OpenMLDB* folder
+Step 3: [Start the cluster](https://openmldb.ai/docs/zh/main/deploy/install_deploy.html) and enter the *OpenMLDB* folder
```

Comment:
Add a simpler way to start cluster, https://openmldb.ai/docs/zh/main/quickstart/openmldb_quickstart.html#id4, and note that it's only for test, for the beginner, the performance is not good cuz it's just an onebox. But it's hard for beginners to deploy a distribute cluster.